app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentMain.kt

```diff
@@ -0,0 +1,71 @@
+package ar.scacchipa.twittercloneapp.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.activity.addCallback
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import ar.scacchipa.twittercloneapp.R
+import ar.scacchipa.twittercloneapp.databinding.FragmentMainBinding
+import java.util.*
+
+class FragmentMain : Fragment() {
```

Comment:
This file is not need it, you are not managing the navigation correctly. The main is the activity which you already had and from there you navigate through the different fragments